coding-standard-baseline/action.yml

```diff
@@ -0,0 +1,129 @@
+    default: "8.1"
+    description: "PHP version used to do the coding standard check."
+
+  composer_version:
+    type: string
+    required: true
+    default: "2"
+    description: "The version of composer to use."
+
+  path:
+    type: string
+    required: true
+    default: 'app/code'
+    description: "The directory (relative to the project root) in which the coding standard will be checked. Used when the event is not a pull request."
+
+  version:
+    type: string
+    required: false
+    default: "31"
+    description: "The version of the coding standard to use. If not provided, will use the latest version."
+
+  severity:
+    type: string
+    required: false
+    default: "8"
+    description: "The minimum severity required to display an error or warning (default: 5)"
+
+  warning_severity:
+    type: string
+    required: false
+    default: "8"
+    description: "The minimum severity required to display a warning"
+
+  error_severity:
+    type: string
+    required: false
+    default: "8"
+    description: "The minimum severity required to display an error"
+
+  baseline_version:
+    type: string
+    required: false
+    default: "1.1.2"
+    description: "version of phpcs baseline"
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout Project
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Set PHP Version
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ inputs.php_version }}
+        tools: composer:v${{ inputs.composer_version }}
+        coverage: none
+
+    - name: Install Coding Standard && Codesniffer baseline
+      shell: bash
+      run: |
+        git config --global advice.detachedHead false
+        composer require "magento/magento-coding-standard: ${{ github.event.inputs.version || '*' }}" -W
+        composer config --no-plugins allow-plugins.digitalrevolution/php-codesniffer-baseline true
+        composer require --dev "digitalrevolution/php-codesniffer-baseline: ${{ github.event.inputs.baseline_version || '*' }}"
+
+    - name: Register Coding Standard
+      shell: bash
+      run: vendor/bin/phpcs --config-set installed_paths ${{ github.workspace }}/vendor/magento/magento-coding-standard,${{ github.workspace }}/vendor/phpcompatibility/php-compatibility
+
+    - name: Get all changed files
+      id: changed-files
+      uses: tj-actions/changed-files@v35
+      with:
+        write_output_files: true
+        output_dir: ${{ github.workspace }}
+        separator: " "
+
+    - name: Verify the contents of the modified_files.txt file
+      shell: bash
+      run: |
+        sed "s/ /\n/g" ${{ github.workspace }}/modified_files.txt > ${{ github.workspace }}/newline_file.txt
+        grep -iE "\.php|\.phtml" ${{ github.workspace }}/newline_file.txt > ${{ github.workspace }}/phpcs_files.txt
+        cat ${{ github.workspace }}/phpcs_files.txt
+
+    - name: Checkout - Before Merge
+      shell: bash
+      id: previous-commit-files
```

Comment:
Since we do not use these `id`s, should we just remove them from all steps?

Reply:
sure will remove that

Reply:
this is done